### DISCUSSION TITLE
Quasar QStepper Component can't use dynamic captions

### DISCUSSION THREAD

**Discussion by karimPortfolio** [quasarframework/quasar#17712] [General - Components / Directives / etc]
Hey Quasar Team,
First  i wanna thank you for this awesome framework, it helps me and my team so much to build new features quickly and with high quality. QStepper is the most useful component for us the only problem is when we need to add a html tags to caption which doesn't have a slot, QStepper really needs to have a caption slot for cases when you need a custom caption.

**Comment by Adnane-elKanbouji** [comment]
I faced the same issue Quasar defintely need to add QStepper slot and a horizontaly scroll area for Qtable.

**Comment by OtmaneKhalil** [comment]
Me and my team we had same problem USING Qstepper can't customize Qstepper caption, Quasar team need to add slot for Qtepper caption.
